src/edu/psu/sweng500/team8/gui/CustomKeyListener.java

```diff
@@ -24,7 +24,10 @@ public CustomKeyListener(Cell cell, GameSession gameSession) {
 
 	public void keyTyped(java.awt.event.KeyEvent keyEvent) {
 		
-		if (KeyEvent.VK_BACK_SPACE == keyEvent.getKeyChar()) {
+		
+		
+		if (KeyEvent.VK_BACK_SPACE == keyEvent.getKeyChar() || KeyEvent.VK_DELETE == keyEvent.getKeyChar()) {
+			
 			gameSession.enterNumber(this.cell, 0);
 		} else {
 			gameSession.enterNumber(this.cell, Integer.parseInt(String.valueOf(keyEvent.getKeyChar())));

```

Comment:
I can't see a problem, so please give your code on how to handle this
otherwise.

On Tue, Jul 7, 2015 at 7:44 AM, Jeremy Nelson <notifications@github.com>
wrote:

> In src/edu/psu/sweng500/team8/gui/CustomKeyListener.java
> <https://github.com/garnryang/Team8/pull/10#discussion_r34031037>:
>
> > @@ -24,7 +24,10 @@ public CustomKeyListener(Cell cell, GameSession gameSession) {
> >
> >  	public void keyTyped(java.awt.event.KeyEvent keyEvent) {
> >  		
> > -		if (KeyEvent.VK_BACK_SPACE == keyEvent.getKeyChar()) {
> > +		
> > +		
> > +		if (KeyEvent.VK_BACK_SPACE == keyEvent.getKeyChar() || KeyEvent.VK_DELETE == keyEvent.getKeyChar()) {
> > +			
> >  			gameSession.enterNumber(this.cell, 0);
> >  		} else {
> >  			gameSession.enterNumber(this.cell, Integer.parseInt(String.valueOf(keyEvent.getKeyChar())));
>
> Regardless of whether it works right now or not, it's not good to have
> order-dependent code separated like this.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/garnryang/Team8/pull/10/files#r34031037>.
>